stubs/inertia/resources/js/Components/Dropdown.vue

```diff
@@ -6,9 +6,9 @@ const props = defineProps({
         type: String,
         default: 'right',
     },
-    width: {
+    widthClass: {
```

Comment:
This isn't really width-specific any longer, so I wonder whether `contentClass` would be more appropriate. It's effectively just exposing the `class` attribute for the `content` slot.